README.md

```diff
@@ -96,7 +86,6 @@ Here is the launch configuration generated for Node.js debugging
             "env": {
                 "APS_CLIENT_ID": "your clientid here",
                 "APS_CLIENT_SECRET": "your client secret here",
```

Comment:
Btw the trailing comma here will probably invalidate the JSON, right?

Reply:
Just checked, I have that in another project as well, and VS Code does not seem to mind. I think it used to be more fussy about it in the past.